x/nft/client/cli/cli_test.go

```diff
@@ -29,11 +35,21 @@ type IntegrationTestSuite struct {
 	network *network.Network
 }
 
+func GetApp(val network.Validator) servertypes.Application {
```

Comment:
There are two same`GetApp` in the different tests. perhaps can move it to `testutil/test_helpers.go`